kyo-prelude/shared/src/main/scala/kyo/Ack.scala

```diff
@@ -0,0 +1,66 @@
+package kyo
```

Comment:
,oved to the `kyo` package since it's shared between `Emit`, `Poll`, and `Stream`.